scripts/blackbox_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script installs Web3 from virtual registry
+# env and runs some simple Web3 calls.
+
+# Exit immediately on error
+set -o errexit
+
+echo "Node version"
+node --version
+
+# Setup mock project to install web3 from virtual registry
+mkdir blackbox_test_app
+cp scripts/js/basic_usage.ts blackbox_test_app/basic_usage.ts
+cp -r fixtures blackbox_test_app/.
```

Comment:
I would suggest to keep one directory for the test app with all the files and `package.json` in it. And just copy that directory. 

This will be easy to extend that app with more behavior in it and cover more edge cases during the development.

Reply:
sounds good. created a`blackbox_test` folder